research/GPT/recipes/90sparse.yaml

```diff
@@ -0,0 +1,152 @@
+version: 1.1.0
+
+modifiers:
+    - !EpochRangeModifier
+        end_epoch: 10
+        start_epoch: 0.0
+
+        - !GMPruningModifier
```

Comment:
is there a reason that there are so many `GMPruningModifier`s here? it looks like they generally all have the same hyperparameters